docs/source/getting-started/assessing-utility/accuracy.ipynb

```diff
@@ -0,0 +1,374 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accuracy Method"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Many prior examples of queries used comparisons of the actual values to the computed differentially private values. Intuitively, we assessed the quality of our results by visually inspecting the proximity of the actual values to the differentially private outputs. Moreover, comparing one DP release to the true value is a comparison with exactly 1 draw from a random variable. One may be \"lucky\" and get a release that is much more accurate than usual, or \"unlucky\" and get a release that is much less accurate than usual.\n",
+    "\n",
+    "However, this approach isn't concrete since interpretations can vary widely on scale, type of visualization, and other factors. A more objective way to evaluate differentially private outputs is by using the `accuracy` method. \n",
+    "\n",
+    "With synthetic data a comparison with the \"true\" value is possible. However, this is not possible with real data. If the analysis plan is influenced by the actual data, it's a privacy leak. \n",
+    "\n",
+    "\n",
+    "In this notebook, we will: \n",
+    "- Explain the purpose and the application of the `accuracy` method. \n",
+    "- Clarify how to interpret the results of the `accuracy` method. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import opendp.prelude as dp\n",
+    "import polars as pl \n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np \n",
+    "import pandas as pd \n",
+    "\n",
+    "dp.enable_features(\"contrib\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
```

Comment:
As before, enclose this in a box, whilst stating what this is to make things cleaner.

Reply:
There is a comment: 
```
# Fetch and download data
```
Maybe we'd just like to hide this output cell?